src/transformers/models/auto/tokenization_auto.py

```diff
@@ -170,6 +170,7 @@
     GPT2TokenizerFast = None
     HerbertTokenizerFast = None
     LayoutLMTokenizerFast = None
+    LEDTokenizerFast = None
```

Comment:
This was actually recently corrected in another commit this morning - so there might be merge conflicts, but should be easy to resolve